src/ngrx/11-create-reducer/create-reducer.md

```diff
@@ -9,21 +9,32 @@
 
 ## Overview
 
-1. Define Login state shape
+1. Define Login feature state shape
```

Comment:
Is "Feature State" preferable to "Feature Set", or is this a different concept?

Reply:
I don't believe state and store are the same thing:

store: a collection of state, reducer, meta-reducers + selectors + actions (and sometimes effects)

As for set vs store, I'm having a hard time finding any documentation suggesting they're different, but since "set" only comes up for generating features:

set: a subset of a (global) store that is specific to one feature (I think...)